app/build.gradle

```diff
@@ -2,8 +2,9 @@ plugins {
     alias(libs.plugins.android.application)
 
     // Add the Google services Gradle plugin
-    id 'com.google.gms.google-services'
-    id 'com.google.firebase.crashlytics'
+    // Comment out Firebase plugins temporarily
+    // id 'com.google.gms.google-services'
+    // id 'com.google.firebase.crashlytics'
```

Comment:
Revert these lines before merge (just for my own testing so I didn't have to set up Google Services / Firebase).